dist/.eslintrc.json

```diff
@@ -1,9 +1,22 @@
 {
-  "root": true,
+	"root": true,
 
-  "extends": "../.eslintrc-browser.json",
-  "rules": {
-	// That is okay for the built version
-	"no-multiple-empty-lines": "off"
-  }
+	"extends": "../.eslintrc-browser.json",
+	"rules": {
+		// That is okay for the built version
+		"no-multiple-empty-lines": "off",
+
+	  // Sizzle is not compatible with jQuery code style
+	  "no-nested-ternary": "off",
```

Comment:
A general remark - why do we have to disable all of these rules for `dist` when we don't do that for `src` and `sizzle.js` is just copied to `dist`? That shouldn't be necessary.

(You'll need to run `npm run build` & commit the changed files in `dist`, though)

Reply:
It was my mistake, I didn't commit my `dist` files and travis job failed. Now I have commit all the files all this configuration should be not needed.